docs/source/user-guide/sql/window_functions.md

```diff
@@ -148,28 +148,10 @@ All [aggregate functions](aggregate_functions.md) can be used as window function
 
 ## Analytical functions
 
-- [cume_dist](#cume_dist)
-- [percent_rank](#percent_rank)
 - [first_value](#first_value)
 - [last_value](#last_value)
 - [nth_value](#nth_value)
 
-### `cume_dist`
```

Comment:
I double checked the functions are documented here: https://datafusion.apache.org/user-guide/sql/window_functions_new.html#cume-dist